subprojects/hashing/src/test/groovy/org/gradle/internal/hash/HashCodeTest.groovy

```diff
@@ -16,58 +16,48 @@
 
 package org.gradle.internal.hash
 
+import org.gradle.internal.hash.HashCode.ByteArrayBackedHashCode
+import org.gradle.internal.hash.HashCode.HashCode128
 import spock.lang.Specification
 
 class HashCodeTest extends Specification {
     def "can parse hex string #input"() {
         def hash = HashCode.fromString(input)
 
         expect:
-        hash.toString() == toString
-        hash.toByteArray() == bytes
-        hash.length() == length
-        hash.hashCode() == (int) hashCode
-
-        where:
-        input          | length | toString       | hashCode   | bytes
-        "12345678"     | 4      | "12345678"     | 0x78563412 | toBytes(0x12, 0x34, 0x56, 0x78)
-        "CAFEBABE"     | 4      | "cafebabe"     | 0xBEBAFECA | toBytes(0xCA, 0xFE, 0xBA, 0xBE)
-        "abbaabba"     | 4      | "abbaabba"     | 0xBAABBAAB | toBytes([0xAB, 0xBA] * 2)
-        "abbaabbaabba" | 6      | "abbaabbaabba" | 0xBAABBAAB | toBytes([0xAB, 0xBA] * 3)
-        "aB" * 255     | 255    | "ab" * 255     | 0xABABABAB | toBytes([0xAB] * 255)
-    }
-
-    def "can parse int: #input"() {
```

Comment:
Should the `can parse int` tests go to a `TestHashCode(s)Test`?

Reply:
I don't think those are useful or needed anymore.